docs/plugin-protocol/tfplugin5.proto

```diff
@@ -811,3 +819,45 @@ message CallFunction {
         FunctionError error = 2;
     }
 }
+
+message ListResource {
+    message Request {
+        // type_name is the managed resource type name.
+        string type_name = 1;
+
+        // configuration is the list ConfigSchema-based configuration data.
+        DynamicValue config = 2;
+    }
+
+    message Event {
+        // Resource contains all data associated with each resource instance.
+        message Resource {
```

Comment:
Should we rename this to `Result` or `ManagedResource` ? 
`Resource` is such an overloaded name, and I think it may be confusing.

Reply:
I'm in favor of 'Result' then, because 'ManagedResource` implies to me that it is a resource managed by Terraform, which isn't the case here.